### ISSUE TITLE
Custom data label not shown

### ISSUE THREAD

**Issue by andrea689** [syncfusion/flutter-examples#159]
Hi,
is there a way to show custom data label on top of chart point instead of in the middle?
My problem is that when the column is too short, the label is not shown.

With simple data label is ok:
<img width="342" alt="Screenshot 2020-07-06 at 11 48 44" src="https://user-images.githubusercontent.com/5927329/86580554-2de5ad00-bf7f-11ea-9794-5db555a2a7a1.png">

With custom data label is not shown:
<img width="344" alt="Screenshot 2020-07-06 at 11 48 28" src="https://user-images.githubusercontent.com/5927329/86580558-2f16da00-bf7f-11ea-8e18-65be5b4dcc44.png">

**Comment by SriramKiranSenthilkumar** [comment]
Hi @andrea689 ,

Greetings from Syncfusion. We have analyzed your scenario and we were able to replicate the reported issue regarding the custom data label not showing when y-value is too short for the data points. We will consider this issue as a bug and will include the fix for the reported issue in our upcoming patch release which is expected to be rolled out by the middle of next week (14th July 2020). We will update you once the release is rolled out and we appreciate your patience until then.

Regards,
Sriram Kiran.
